src/modules/student-import/student-import.controller.ts

```diff
@@ -7,11 +7,14 @@ import {
 } from '@nestjs/common';
 import { StudentImportService } from './student-import.service';
 import { FileInterceptor } from '@nestjs/platform-express';
+import { Roles } from 'src/common/decorators/roles.decorator';
+import { Role } from 'src/types';
 
 @Controller('/import')
 export class StudentImportController {
   constructor(private readonly studentImportService: StudentImportService) {}
 
+  @Roles(Role.ADMIN)
   @Post('/students/')
```

Comment:
Czy tu nie jest o jeden ukośnik za dużo? 
czy nie powinno być po prostu:
@Post('/students')
?
w innych miejscach też to widzę, większych problemów to chyba nie tworzy w działaniu... ale kto wie, może kiedyś to zawadzić, więc może lepiej bez tego ukośnika na końcu?

Reply:
poprawione